data_layer/damock/damock.go

```diff
@@ -13,7 +12,7 @@ type DAMock struct {
 }
 
 func (d *DAMock) GetStatus(c config.RollappConfig) string {
-	return ""
+	return "Running mock DA"
```

Comment:
why

Reply:
the DA row is displayed now, u prefer something else to be written for mock DA?